baybe/utils/dataframe.py

```diff
@@ -55,10 +55,12 @@ def to_tensor(*dfs: pd.DataFrame) -> Tensor | Iterator[Tensor]:
     from baybe.utils.torch import DTypeFloatTorch
 
     out = (
```

Comment:
that's a generator comprehension, so typing wise everything is fine, I think. but not sure if that's what you meant? 
Anyway, the choice for generator was sort of arbitrary. In fact, not much is won here since the inputs come in as tuple anyway 🙈 so avoiding explicit collection via generator brings no benefits because the input is automatically pre-collected.

If no complaints from your side, would thus change it to an actual tuple

Reply:
changed